### PR TITLE
Tweak Lolex definitions

### PR DESCRIPTION
**Pull request by SimenB**
A few tweaks from trying to use this definition.

/cc @marudor @fatso83 @benjamingr

### CODE REVIEW

File: definitions/npm/lolex_v2.x.x/flow_v0.64.x-/lolex_v2.x.x.js [hunk 1]
```diff
@@ -35,8 +37,8 @@ declare module 'lolex' {
     runMicrotasks(): void;
     runToFrame(): void;
     runToLast(): void;
-    setSystemTime(now?: Date): void;
-    uninstall(): void;
+    setSystemTime(now?: number | Date): void;
```
Comment (SimenB) [inline definitions/npm/lolex_v2.x.x/flow_v0.64.x-/lolex_v2.x.x.js]: https://github.com/sinonjs/lolex/blob/d4c2c053c8dce7d8b9e6727ac75275e5e32bae2b/src/lolex-src.js#L107-L116

File: definitions/npm/lolex_v2.x.x/flow_v0.64.x-/lolex_v2.x.x.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ declare module 'lolex' {
   declare type lolex = {
     createClock(now?: number, loopLimit?: number): Clock,
     install(config?: installConfig): Clock,
+    timers: Object,
```
Comment (SimenB) [inline definitions/npm/lolex_v2.x.x/flow_v0.64.x-/lolex_v2.x.x.js]: https://github.com/sinonjs/lolex/blob/d4c2c053c8dce7d8b9e6727ac75275e5e32bae2b/src/lolex-src.js#L860

File: definitions/npm/lolex_v2.x.x/flow_v0.64.x-/lolex_v2.x.x.js [hunk 3]
```diff
@@ -35,8 +37,8 @@ declare module 'lolex' {
     runMicrotasks(): void;
     runToFrame(): void;
     runToLast(): void;
-    setSystemTime(now?: Date): void;
-    uninstall(): void;
+    setSystemTime(now?: number | Date): void;
+    uninstall(): Object[];
```
Comment (SimenB) [inline definitions/npm/lolex_v2.x.x/flow_v0.64.x-/lolex_v2.x.x.js]: https://github.com/sinonjs/lolex/blob/d4c2c053c8dce7d8b9e6727ac75275e5e32bae2b/src/lolex-src.js#L426-L432

Comment (SimenB) [reply definitions/npm/lolex_v2.x.x/flow_v0.64.x-/lolex_v2.x.x.js]: (might wanna type the `timers` object better)

File: definitions/npm/lolex_v2.x.x/flow_v0.64.x-/lolex_v2.x.x.js [hunk 4]
```diff
@@ -25,7 +26,8 @@ declare module 'lolex' {
     cancelAnimationFrame: typeof cancelAnimationFrame;
     hrtime: typeof process.hrtime;
     nextTick: typeof process.nextTick;
-    performace: {
+    now: number;
```
Comment (SimenB) [inline definitions/npm/lolex_v2.x.x/flow_v0.64.x-/lolex_v2.x.x.js]: https://github.com/sinonjs/lolex/blob/d4c2c053c8dce7d8b9e6727ac75275e5e32bae2b/src/lolex-src.js#L521

File: definitions/npm/lolex_v2.x.x/flow_v0.64.x-/lolex_v2.x.x.js [hunk 5]
```diff
@@ -25,7 +26,8 @@ declare module 'lolex' {
...
+    performance?: {
```
Comment (SimenB) [inline definitions/npm/lolex_v2.x.x/flow_v0.64.x-/lolex_v2.x.x.js]: not sure about this one - but performance is not there unless it exists in the `global` it installs from. So I _think_ adding the `?` is correct.

**Comment by fatso83** [comment]
seems legit.

**Pull request merged**